src/Components/Column/Column.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes  from "prop-types"
+import { styled } from "@mui/system"
+
+export const Column = ({children}) =>{
```

Comment:
add `dataTestId` ..add for other components as well ..

Reply:
done